core/src/main/java/org/apache/accumulo/core/fate/accumulo/AccumuloStore.java

```diff
@@ -112,6 +121,16 @@ public FateId getFateId() {
     return FateId.from(fateInstanceType, tid);
   }
 
+  @Override
+  protected void create(FateId fateId, FateKey fateKey) {
+    var status = newMutator(fateId).requireStatus().putStatus(TStatus.NEW).putKey(fateKey)
+        .putCreateTime(System.currentTimeMillis()).tryMutate();
+
+    // TODO: Any reason to retry here?
```

Comment:
An unknown status means that a failure happened w/ the RPC for the conditional mutation and its not know if it went through.  So need to scan in that case and see if it was created or not and possibly retry.